### DISCUSSION TITLE
RC Question type for creation of new questions

### DISCUSSION THREAD

**Discussion by Prerna789** [Sunbird-coKreat/Community#4] [General Discussions]
Hi @Krishnaj20 

I am initiating this discussion thread to understand if there are any **UI wireframes and PRD for enabling Reading Comprehension question type** as part of creation of questions workflow.
Could you please tag Pranav from Shiksha Lokam on this thread as I am not able to locate his github ID.
I'll take it forward from there.
Thanks!

**Comment by Krishnaj20** [comment]
@Prerna789 - RC question type is not enabled by the inQuiry building block. 
@Prateek-slokam - Is this question type in your contribution roadmap?

**Reply by Prerna789** [reply]
Yes, I am aware of it, wanted to ask if it is part of Shiksha Lokam's contribution roadmap. 
Thanks for tagging Prateek to the thread.

**Comment by Prateek-slokam** [comment]
@Prerna789 @Krishnaj20 I haven't completely understood what is particularly going to be in the RC type question or what is being imagined there. I would mention the question types that we have: MCQ (both single select and multi select), slider type (like in google forms where a slider value can be set), date type, text type (for free text entry) and number type (a validation/configuration on text type question to have users only be able to fill in numbers).
For the next 2-3 quarters we don't plan to add any other question type than the above mentioned ones.

**Reply by Krishnaj20** [reply]
thanks @Prateek-slokam . Can you share when you are targeting to contribute these new question types (MCQ (both single select and multi select), slider type (like in google forms where a slider value can be set), date type, text type (for free text entry) and number type (a validation/configuration on text type question to have users only be able to fill in numbers)? Also, l believe none of these question types have scoring enabled (right answer/incorrect answer etc). Pls confirm. 
@pallakartheekreddy

**Reply by Prateek-slokam** [reply]
@Krishnaj20 The contribution as I understand is in progress. A few PRs have already been raised for the observation and survey creation tool. These include the question types too. 
Scoring has been enabled as a configurable feature by the creator only for the MCQ question types. The creator can choose to have and not have scoring in observation without rubric. In survey the scoring option is not there while observation with rubric by default has scoring enabled since that is required to define the rubric ranges.

**Comment by Prerna789** [comment]
Hey @Prateek-slokam, ok noted.
Can you share the PRD and mockups for the question types you do have in pipeline, i.e. MCQ, slide type, date type, text type etc
Would be good to go through it once.
Parallely, Samagra team is planning to work on adding RC question type to the portal.

I will share a PRD for it once it gets ready for reference.
Thanks.

**Reply by Prateek-slokam** [reply]
Hey @Prerna789,
The PRD including the explanation of the question types and link to other tech design docs can be found [here](https://project-sunbird.atlassian.net/wiki/spaces/SingleSource/pages/2822963275/Observation+Survey+creation+portal+PRD#New-Question-categories-that-would-be-added). [This](https://project-sunbird.atlassian.net/browse/SB-21820) is the jira story for the question types. Also you can refer to [this figma UI](https://www.figma.com/proto/MgE9UW1rrC4cYfcbO9RO7t/Vidyadaan?node-id=99%3A1630&scaling=min-zoom&page-id=88%3A2) for further understanding of the question types UI.
